modmesh/core.py

```diff
@@ -105,6 +107,34 @@ def _load():
     for name in __all__:
         globals()[name] = getattr(_impl, name)
 
+    # Try to find the thirdparty, if failed to find it
+    # modmesh will raise ImportError and remind the user.
+    filename = os.path.join('thirdparty')
+    path = os.getcwd()
+    try:
+        while True:
+            if os.path.exists(os.path.join(path, filename)):
+                break
+            if path == os.path.dirname(path):
+                path = None
+                break
+            else:
+                path = os.path.dirname(path)
+        if path is None or not os.path.exists(os.path.join(path,
```

Comment:
It is harder to read the lines when they are broken like:

```python
if path is None or not os.path.exists(os.path.join(path,
                                                   filename)):
```

In this case, breaking the line using a local variable is more readable:

```python
_pf = os.path.join(path, filename)
if path is None or not os.path.exists(_pf):
```